### PR TITLE
Settings rename / cleaup

### PR DESCRIPTION
**Pull request by snopoke**
See https://github.com/dimagi/commcare-hq/pull/8893

### CODE REVIEW

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -662,7 +649,7 @@ SMS_PREFIXES_BY_COUNTRY = '{{ localsettings.SMS_PREFIXES_BY_COUNTRY }}'
 {% endif %}
 
 {% if localsettings.get('MIA_THE_DEPLOY_BOT') %}
-MIA_THE_DEPLOY_BOT = '{{ localsettings.MIA_THE_DEPLOY_BOT }}'
+MIA_THE_DEPLOY_BOT_API = '{{ localsettings.MIA_THE_DEPLOY_BOT }}'
```
Comment (benrudolph) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: i think this should be `localsettings.MIA_THE_DEPLOY_BOT_API`

Comment (benrudolph) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: actually that depends if you're planning on changing the name in the `config` dir

**Comment by benrudolph** [comment]
may be worthwhile to also change the var names in the `dev.yml` (and the config dirs) so that there remains a consistency between the two

**Comment by sravfeyn** [comment]
This looks good. This needs to be coordinated with the deploy, right?

**Comment by snopoke** [comment]
I've added back in the old settings as well so that I can deploy this now and merge the HQ PR. Once HQ has been deployed I'll remove the old settings.

**Comment by czue** [comment]
you still want a don't merge label on this?

**Comment by snopoke** [comment]
removed label, thanks

**Pull request merged**